GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/DatabaseTests/Return/ReturnTests.IsLateSubmission.cs

```diff
@@ -23,6 +23,10 @@ public void Is_True_When_ModifiedDate_Is_Late_And_InScope(SectorTypes sector, Sc
             {
                 // Arrange 
                 int testYear = VirtualDateTime.Now.Year - yearOffset;
+                if (Global.ReportingStartYearsToExcludeFromLateFlagEnforcement.Contains(testYear))
+                {
+                    continue;
+                }

```

Comment:
I don't understand what we're trying to achieve here? Could you explain please 😄?

Reply:
To be honest I don't really understand the reason why this test is implemented this way and I should probably refactor it.
**What it does:** It checks if a report is marked as late if submitted after the deadline. We need to test that for a reporting year (but why do we need 4 years?? - no idea).

**Why I added these lines:** Now `ReportingStartYearsToExcludeFromLateFlagEnforcement` is configured as well. Therefore, if one of the years is excluded we should skip it. Otherwise, the test will fail as reports should NOT be marked as late if the year is excluded from late flag enforcement.

**What I should have done:** Update the test to check this behavior for a SINGLE year that's not excluded from late flag enforcement.

Reply:
Sorry, I had completely forgotten about what the continue statement is for 😅.
Your explanation is quite good, thanks!

I'm also a bit confused why they test for 4 years. I don't think it's bad to test for more than one year, but again, I think this should be done as a TestCase, instead of using a for loop (I know this is possibly in python, no idea about C#).

Reply:
ah sorry. `continue;` skips the current iteration of the loop and forces the next one
`break;` also skips the current iteration but it exits the loop instead of going to the next iteration